core/src/main/java/com/coderanch/blackjack/Cards.java

```diff
@@ -61,4 +64,22 @@ private Cards() {}
     static Set<Card> getStandardDeck() {
         return STANDARD_DECK;
     }
+
+    static Stack<Card> getShuffledStandardDeck(){
+        return getStandardDeck().stream()
+                .map(c -> new CardPair(RANDOM.nextInt(10_000_000), c))
+                .sorted((o1, o2) -> Integer.compare(o1.position, o2.position))
+                .map(cp -> cp.card)
+                .collect(Collectors.toCollection(Stack::new));
```

Comment:
There is an easier way to shuffle a collection: Convert the collection to a list, call `Collections.shuffle(list, generator)` and then convert the list back to a stack.